deploy/docker/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.8.7-slim
 
 # Update
 RUN apt update -y && apt upgrade -y
-RUN apt install patch gcc libffi-dev wget -y
+RUN apt install patch gcc libffi-dev wget git -y
```

Comment:
I guess once we stop using your pyumbral branch and have a pyumbral release with those changes, we can remove the need for 'git'.

Reply:
Yep, it's in a separate commit and can be removed as soon as pyumbral 0.2 is out.